File: app/src/main/java/com/song/deviceinfo/ui/usb/USBViewModel.java
package com.song.deviceinfo.ui.usb;

import android.content.Context;

import com.song.deviceinfo.info.USBInfo;
import com.song.deviceinfo.ui.base.NormalViewModel;

import java.util.List;

import androidx.core.util.Pair;

/**
 * Created by chensongsong on 2020/6/8.
 */
public class USBViewModel extends NormalViewModel {

    public List<Pair<String, String>> getUsbInfo(Context context) {
        return USBInfo.getUSBInfo(context);
    }
}

File: app/src/main/java/com/song/deviceinfo/ui/usb/USBViewModel.kt
package com.song.deviceinfo.ui.usb

import android.content.Context
import androidx.core.util.Pair
import com.song.deviceinfo.info.USBInfo
import com.song.deviceinfo.ui.base.NormalViewModel

/**
 * Created by chensongsong on 2020/6/8.
 */
class USBViewModel : NormalViewModel() {
    fun getUsbInfo(context: Context?): List<Pair<String, String>> {
        return USBInfo.getUSBInfo(context!!)
    }
}
